### PR TITLE
Slider README

### LINKED ISSUE #189: Slider: Add Content to README
Please refer to the related Epic for guidance on updating the README.

### PR DESCRIPTION
**Pull request by smhigley**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:**

Resolves #189, adds Slider readme and fixes event typings on the example page.

### CODE REVIEW

File: src/slider/README.md [hunk 1]
```diff
@@ -1 +1,83 @@
-# Slider widget
+# @dojo/widgets/slider/Slider widget
+
+Dojo 2's `Slider` widget creates a range slider control with a styleable track, fill, and thumb.
```
Comment (dylans) [inline src/slider/README.md]: Will everyone know what a thumb is?

File: src/slider/README.md [hunk 2]
```diff
@@ -1 +1,83 @@
...
+
+
+## Features
+
+- Horizontal or vertical slider with a track, fill, and single thumb
+- Output node with customizable text
+
+### Accessibility Features
+
+`Slider` uses the native `<input type="range">`as its base, which ensures built-in keyboard and screen reader accessibility. All common form field attributes (`disabled`, `invalid`, `readOnly`, `required`) may be set, as well as a visible or hidden label. The output node is associated with the input.
```
Comment (dylans) [inline src/slider/README.md]: "The output node is associated with the input." I'm not sure most readers will know what is meant by this.

File: src/slider/README.md [hunk 3]
```diff
@@ -1 +1,83 @@
...
+		});
+	}
+});
+```
+
+## Theming
+
+The following CSS classes are available on the `Slider` widget for use with custom themes:
+
+- `root`: Applied to either the wrapping `<label>`, or a `<div>` in the same position in the node hierarchy
```
Comment (dylans) [inline src/slider/README.md]: Not essential but it might be nice to organize these into two sections (always, vs. conditional)?

Comment (smhigley) [inline src/slider/README.md]: Done 👍

**Pull request merged**